Source/dom/layers/SymbolMaster.js

```diff
@@ -102,6 +102,8 @@ SymbolMaster[DefinedPropertiesKey] = { ...Artboard[DefinedPropertiesKey] }
 Factory.registerClass(SymbolMaster, MSSymbolMaster)
 Factory.registerClass(SymbolMaster, MSImmutableSymbolMaster)
 
+delete SymbolMaster[DefinedPropertiesKey].background
```

Comment:
Why don't we expose the Background color for Symbol Masters? You can set it on the UI, and include it in Export and in Instances… 🤔

Reply:
Oh you can! I was looking on the symbol instance 🙈